symengine/printer.cpp

```diff
@@ -712,7 +712,7 @@ void StrPrinter::bvisit(const MultivariatePolynomial &x)
                     expr << "*";
                 }
                 expr << it->__str__();
-                if (exps[i] > 1)
+                if (exps[i] > 1 or exps[i] < 0)
```

Comment:
Yes, we want the correction to be in the tests, so that this bug will not appear again.